app/App.hs

```diff
@@ -62,24 +62,29 @@ getEntryPoint' opts root pkg inputFile = do
         _entryPointNoTermination = opts ^. globalNoTermination,
         _entryPointNoPositivity = opts ^. globalNoPositivity,
         _entryPointNoStdlib = opts ^. globalNoStdlib,
-        _entryPointStdlibPath = opts ^. globalStdlibPath,
+        _entryPointStdlibPath = someBaseToAbs root <$> opts ^. globalStdlibPath,
         _entryPointPackage = pkg,
-        _entryPointModulePaths = pure (inputFile ^. pathPath),
+        _entryPointModulePaths = pure (someBaseToAbs root (inputFile ^. pathPath)),
```

Comment:
This code assumes that `inputFile` is either an absolute path or a relative path inside the Juvix project.

However we previously supported running the juvix commands where `inputFile` is a relative path to a Juvix file but the command is not run from within the juvix project.

e.g say the juvix project root is at `/A/B/C` the Juvix file is `/A/B/C/D.juvix` and the user runs `juvix typecheck B/C/D.juvix` (the `PWD` is `/A`).

There's a shellcheck test that catches this.